smartsim/experiment.py

```diff
@@ -42,9 +40,11 @@
 
 from smartsim._core import dispatch
 from smartsim._core.config import CONFIG
+from smartsim._core.control import interval as _interval
```

Comment:
tl;dr its just me being overprotective!

The longer answer is that since we don't define an `__all__` for this module, when you everything, the names of the imports in that module are also imported! So if a user were to do say:
```py
from smartsim.experiment import *
```
they will get `dispatch` and `CONFIG` and `os` and `t` and etc. in `globals()`.

It's _INCREDIBLY_ unlikely that anyone would care, but some style guides I read recommended it so I thought I would try it out. Totally willing to revert though since it kinda breaks the pattern currently established in SmartSim!